cicd/gitops/configuring-sso-for-argo-cd-using-keycloak.adoc

```diff
@@ -10,6 +10,7 @@ After the {gitops-title} Operator is installed, Argo CD automatically creates a
 
 .Prerequisites
 * Red Hat SSO is installed on the cluster.
+* {gitops-title} Operator is installed on the cluster.
```

Comment:
Hello @varshab1210 after the discussion with  @shipsing ,  we thought of adding Red Hat OpenShift GitOps Operator being installed to the prerequisites, as it is mentioned in the first sentence of the section:

`After the {gitops-title} Operator is installed, Argo CD automatically creates ...`

We believe it would add better clarity.  WDYT?